xl/xdg.py

```diff
@@ -60,6 +60,8 @@
 if config_dirs is None:
     if sys.platform == 'win32':
         config_dirs = [exaile_dir]
+    elif 'freebsd' in sys.platform:
+        config_dirs = [data_dirs]
```

Comment:
I think it should be moved to `/usr/local/etc/xdg/exaile`, but if FreeBSD doesn't like that path then I guess we have no choice.